Cargo.toml

```diff
@@ -113,3 +113,9 @@ lto = 'thin'
 panic = 'abort'
 codegen-units = 1
 
+[patch.crates-io]
+equihash = { git = "https://github.com/zcash/librustzcash.git", rev = "2530eb3a9dfe7164d33e9deaebf26a59d058f38b" }
```

Comment:
I did not check this hash.

Reply:
This corresponds to https://github.com/zcash/librustzcash/pull/794 and will be removed once `zcash_primitives 0.10.2` is published.